Gallery website/fun.js

```diff
@@ -0,0 +1,30 @@
+var slideIndex = 1;
+showSlides(slideIndex);
+
+// Next/previous controls
+function plusSlides(n) {
+  showSlides(slideIndex += n);
+}
+
+// Thumbnail image controls
+function currentSlide(n) {
+  showSlides(slideIndex = n);
+}
+
+function showSlides(n) {
```

Comment:
You can make your code a little clearer by adding more comments to showSlide function by explaining what the n parameter is doing and what each if statement and for loop is doing.